website/content/configuration/_advanced_l2_configuration.md

```diff
@@ -68,3 +68,39 @@ In this way, all the IPs coming from `first-pool` will be reacheable only via `N
 and `NodeB`, and only one of those node will be choosen to expose the IP.
 
 On the other hand, IPs coming from `second-pool` will be exposed always via `NodeC`.
+
+### Specify network interfaces that LB IP can be announced from
+
+In L2 mode, by default a metallb speaker will announce LB IP from all network interfaces of a node. We can use `interfaces` in `L2Advertisement` to select a subset of them.
+
+```yaml
+apiVersion: metallb.io/v1beta1
+kind: L2Advertisement
+metadata:
+  name: example
+spec:
+  ipAddressPools:
+  - third-pool
+  interfaces:
+  - eth0
+  - eth1
+```
+
+By this `L2Advertisement`, LB IPs from `third-pool` will be only announced from interfaces `eth0` and `eth1` of all nodes.
+
+```yaml
+apiVersion: metallb.io/v1beta1
+kind: L2Advertisement
+metadata:
+  name: example
+spec:
+  ipAddressPools:
+  - fourth-pool
+  nodeSelectors:
+  - matchLabels:
+      kubernetes.io/hostname: NodeA
+  interfaces:
+  - eth3
+```
+
+`interfaces` can also be used together with `nodeSelectors`. IPs from `fourth-pool` will be announced only from `eth3` of `NodeA`.
```

Comment:
```suggestion
The `interfaces` selector can also be used together with `nodeSelectors`. In this example, the IPs from `fourth-pool` will be announced only from `eth3` of `NodeA`:
```

I'd flip this, making the statement before the example.

Reply:
ok